src/CoreWCF.Http/src/CoreWCF/HttpClientCredentialType.cs

```diff
@@ -28,7 +29,8 @@ internal static bool IsDefined(HttpClientCredentialType value)
                 value == HttpClientCredentialType.Ntlm ||
                 value == HttpClientCredentialType.Windows ||
                 value == HttpClientCredentialType.Certificate ||
-                value == HttpClientCredentialType.InheritedFromHost);
+                value == HttpClientCredentialType.InheritedFromHost ||
+                value == HttpClientCredentialType.Custom);
```

Comment:
I don't think we need to introduce Custom, I think InheritedFromHost is sufficient here. In WCF that means whatever is configured in IIS for the application is what gets used. That's very similar in concept to whatever is configured in asp.net core is used.